unified-runtime/cmake/helpers.cmake

```diff
@@ -205,21 +205,37 @@ function(add_ur_library name)
         target_link_options(${name} PRIVATE
             $<$<STREQUAL:$<TARGET_LINKER_FILE_NAME:${name}>,link.exe>:LINKER:/DEPENDENTLOADFLAG:0x2000>
         )
+        set_target_properties(${name} PROPERTIES DEBUG_POSTFIX d)
```

Comment:
The caveat of using this is that `urinfo.exe` still wants to load `ur_loader.dll` but can't find it since the file is named `ur_loaderd.dll` which is not ideal.

Reply:
Changed this to `OUTPUT_NAME ${name}d` and `urinfo.exe` is working.